Companions/Brutus/BrutusDialogue.cs

```diff
@@ -84,23 +84,23 @@ public override string NormalMessages(Companion companion)
                 {
                     Mes.Add("*I'm not a fool, I'm certain that what [gn:1] does to [gn:2] is just to bully him. I hope he has an iron will, or else...*");
                     Mes.Add("*Do you think I should extend my bodyguard job to [gn:2] too? I don't like what [gn:1] does to him. But I don't want her to think badly of me, that could hurt my chan- I mean, my professionalism. What should I do?*");
-                    Mes.Add("*Earlier this day, [gn:2] tried to hire me to give a lesson to [gn:1]. I refused it for... Some reason.*");
+                    Mes.Add("*Earlier this day, [gn:2] tried to hire me to teach a lesson to [gn:1]. I refused it for... Some reason.*");
                 }
             }
             if (CanTalkAboutCompanion(3))
             {
-                Mes.Add("*You got a zombie on your town? How were you able to do that? I mean... Wow!*");
+                Mes.Add("*You got a zombie on your team? How were you able to do that? I mean... Wow!*");
```

Comment:
Was actually meant to be town..

Reply:
Sorry I should have guessed that from the rest: would work as: in your town